css/main.css

```diff
@@ -0,0 +1,459 @@
+}
+
+.weather {
+  background-image: url('../media/png/weather.png');
+}
+
+.conference {
+  background-image: url('../media/png/conference.png');
+}
+
+.katchy {
+  background-image: url('../media/png/katchy.png');
+}
+
+.project-stacks {
+  display: flex;
+  flex-wrap: wrap;
+  grid-gap: 15px;
+  padding: 20px 0;
+}
+
+.project-stacks li {
+  display: inline;
+  border: 1px solid #8993a4;
+  padding: 10px 12px;
+  background: transparent;
+  color: #fff;
+  transition: 350ms;
+  cursor: pointer;
+}
+
+.project-stacks li:hover {
+  background: rgba(0, 0, 0, 0.349);
+}
+
+.about-me,
+.about-me-section {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+.about-me-text {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+}
+
+.link {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  padding: 12px;
+  background: #ff6b00;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #fff;
+  align-self: start;
+  transition: 350ms;
+}
+
+.link:hover {
+  background: #a04300;
+}
+
+.about-me-icon {
+  position: absolute;
+  right: 0;
+  bottom: 0;
+}
+
+.list-title {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.languages {
+  list-style: inside url('../media/svg/rectangle.svg');
+}
+
+.frameworks {
+  list-style: inside url('../media/svg/square.svg');
+}
+
+.skills {
+  list-style: inside url('../media/svg/circle.svg');
+}
+
+/* Footer */
+.form-container {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 0;
+  margin: 0 auto;
+  max-width: 85%;
+}
+
+.form-container::after {
+  position: absolute;
+  content: '';
+  height: 1px;
+  left: -9%;
+  right: -9%;
+  bottom: 0;
+  background-color: #dfe1e6;
+}
+
+.contact-msg {
+  color: #172b4d;
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+}
+
+.form {
+  width: 100%;
+  margin-top: 50px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 24px;
+}
+
+.form-control {
+  width: 100%;
+}
+
+.form-control input {
+  border: none;
+  border-bottom: 1px solid #dbd8d7;
+  padding: 15px 12px;
+  font-size: 1rem;
+}
+
+.form-control input::placeholder {
+  font-size: 15px;
+  color: #979493;
+}
+
+.form-control textarea {
+  padding: 12px;
+  background-color: #fbf8f7;
+  border: none;
+  color: #3c3a39;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  height: 114px;
+}
+
+.form-control textarea::placeholder {
+  color: #3c3a39;
+  font-family: 'Inter', sans-serif;
```

Comment:
- [ ] OPTIONAL: To make your CSS code DRY, you can wrap some properties in a call instead of repeating them multiple times in different classes. You can apply the suggestion below:

🔴 **Bad** 
```
  .form-control inputs {
     font-family: 'inter', sans-serif
     /*other properties*/
  }
  .form-control button {
     font-family: 'inter', sans-serif
     /*other properties*/
  }
  .form-control textarea {
     font-family: 'inter', sans-serif
     /*other properties*/
  }
```

✅ **Good**
```
  .text-inter-font {
    font-family: 'inter', sans-serif
  }
  .form-control inputs {
     /*other properties*/
  }
  .form-control button {
     /*other properties*/
  }
  .form-control textarea {
     /*other properties*/
  }
```